app/controllers/apis/test_controller.rb

```diff
@@ -0,0 +1,11 @@
+# api module
```

Comment:
Was it intentional to commit this file? Why do we need this, a spec should be good to test the ApiController class.